themes/default/content/blog/azure-container-solutions/index.md

```diff
@@ -0,0 +1,182 @@
+
+Applications with smaller and more predictable usage patterns are a great use case for ACI. The lack of horizontal scaling could be seen as a negative, but from another perspective, it means the use case is straightforward and simple. If you just want to get your application into the hands of your users and think you can accurately predict your usage patterns, choose ACI.
+
+## Azure App Service
+
+Azure App Service is a more fully-featured offering for hosting applications, focusing more on WebApps and API services. It started life as several differing offerings (Azure Websites, Azure Mobile Service, and Azure BizTalk) before being bundled into a single service in March 2015. Its origins as multiple solutions means it has developed tight integrations with other Azure solutions and offers many value-add features depending on your use case.
+
+Because of the myriad features Azure App Service offers, running containers is only a small part of what you can do with it.
+
+With that in mind, let's take a look at where it shines with regard to running containers.
+
+### What’s it good for?
+
+Azure App Services is designed to be a one-stop shop for your web application needs, so it's highly opinionated and pushes you in a certain direction.
+
+#### Integrations
+
+Azure App Service has many integrations with Azure. As an example, if you need to add a CDN to your web service, it’s extremely straightforward to do so with Azure CDN.
+
+This tight integration with Azure’s many other services makes it likely to meet the needs of your application and extend it as needed, such as if you are introducing common architecture components like message queues and databases.
+
+#### Scaling & Availability
+
+Azure App Service has a myriad of available scaling capabilities built out of the box. Not only can you scale the actual container you’ve deployed (while also simultaneously placing pricing caps on your service), but you can also scale related resources if necessary, which fits well with the tight integration we mentioned previously.
+
+App service scaling for your container can either scale up (ie: increase the amount of resources associated with your container) or scale out (ie: have more containers behind your load balancer).
+
+This flexibility in scaling means you can be safe to assume your application will meet spiking traffic requirements and service your users’ needs.
+
+### What’s not great about it?
+
+#### Opinionated
+
+Azure App Service is highly opinionated. Its tight integrations with other Azure services mean that you're expected to follow established patterns for achieving your goals.
+
+This situation even extends to the allowed languages that Azure App Service supports: If you’re developing your application in an unsupported language, you can’t use the service.
```

Comment:
Why? You can build a container image for any language and put it into App Service.